### PR TITLE
Chore/eslint rules

### PR DESCRIPTION
**Pull request by mahboubii**
for a quick review, main non-auto fixes are here https://github.com/GetStream/stream-js/commit/0fa30aa6040f3a3905ba7e42037e3410bf1043e6

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -10,7 +10,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        node: [10, 11, 12, 13, 14]
+        node: [10, 12, 13, 14]
```
Comment (ferhatelmas) [inline .github/workflows/ci.yml:13]: maybe better to keep 11 here since #314 already adresses

Comment (mahboubii) [reply .github/workflows/ci.yml:13]: yarn install won't work with node v11 which breaks the CI

Comment (mahboubii) [reply .github/workflows/ci.yml:13]: but tests are still passing on v11 so not a breaking change

Comment (ferhatelmas) [reply .github/workflows/ci.yml:13]: > yarn install won't work with node v11 which breaks the CI

isn't it supported anymore in 11 ?

Comment (ferhatelmas) [reply .github/workflows/ci.yml:13]: yup, seems so.

**Pull request merged**